error[E0308]: mismatched types
  --> tests/compile_fail/complete_wrong_second_arg.rs:12:44
   |
12 |     async fn test_prompt(&self, #[complete(complete_wrong_second_arg)] msg: String) -> Result<String> {
   |                                            ^^^^^^^^^^^^^^^^^^^^^^^^^
   |                                            |
   |                                            expected `RequestContext`, found `&mut RequestContext`
   |                                            arguments to this function are incorrect
   |
note: function defined here
  --> tests/compile_fail/complete_wrong_second_arg.rs:5:10
   |
5  | async fn complete_wrong_second_arg(_value: &str, cx: RequestContext) -> Result<Vec<String>> {
   |          ^^^^^^^^^^^^^^^^^^^^^^^^^               ------------------

warning: unused variable: `cx`
 --> tests/compile_fail/complete_wrong_second_arg.rs:5:50
  |
5 | async fn complete_wrong_second_arg(_value: &str, cx: RequestContext) -> Result<Vec<String>> {
  |                                                  ^^ help: if this is intentional, prefix it with an underscore: `_cx`
  |
  = note: `#[warn(unused_variables)]` on by default
